teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorOptions.java

```diff
@@ -140,6 +141,16 @@ public class ValidatorOptions {
   private boolean exitWhenNoValidatorKeysEnabled =
       ValidatorConfig.DEFAULT_EXIT_WHEN_NO_VALIDATOR_KEYS_ENABLED;
 
+  @Option(
+      names = {"--stop-vc-when-validator-slashed"},
```

Comment:
feature flags basically end with '-enabled' - pretty much a standard...
maybe `--stop-vc-when-validator-key-slashed-enabled`

I'd probably expand on the description also...
`If a validator key is detected as slashed, the validator-client should terminate. In this case, the service should not be restarted.`

And we'd want to document how this should all be used pretty carefully.

With systemd, you can stop things from restarting on exit codes ,and i think in the same way, we'd probably want to consider terminating code 2 (rather than code 1 like normal), and then we document that using this feature, you should avoid restarts on code 2 - because if it restarts it'll likely have more slashings. Potentially restarting with doppelganger support would be effective, but definitely would be risky to restart by default...

Reply:
I renamed the option to `--validator-slashing-protection-enabled`, it's clearer imo.
I also updated the description and updated the exit code to 2. I'll update the documentation PR accordingly

Reply:
I tried to describe all the details in a dedicated section in the doc. I raised a draft PR for that: https://github.com/Consensys/doc.teku/pull/526

Reply:
can we not call it `validator-slashing-protection-enabled`? that suggests that without this flag we're not doing local slashing protection...

Reply:
Right, renamed it back to `stop-vc-when-validator-slashed-enabled`